### ISSUE TITLE
Accessibility: Fix Datagrid to work with Screen Reader

### ISSUE THREAD

**Issue by clepore** [infor-design/enterprise#425]
- The current implementation does not work with screen readers. The headers are not properly announced. Solution would be to make the tables visible offscreen or use aria labels to mimic.
- Has invalid aria role grid role on one section

Possibly Revisit http://maxdesign.com.au/jobs/sample-fixed-thead/index.html
Reference: SOHO-7389

**Comment by tmcconechy** [comment]
In 4.24 we added this, the datagrid should work as a table with screen readers now. May need further testing with Ally testers to which we can add new issues from the results.
